README.md

```diff
@@ -1,15 +1,17 @@
 ANYWAY [![Build Status](https://travis-ci.org/hasadna/anyway.png)](https://travis-ci.org/hasadna/anyway)
 ======
 
-oway.org.il - Crowd-sourced road hazard reporting website.
+[anyway.co.il](https://www.anyway.co.il/) - Crowd-sourced road hazard reporting website.
 
-Feel free to contribute to the project.
```

Comment:
I'm not a native English speaker, but I think it should be "to the project"

Reply:
I thought I already changed it... Probably missed it :)
will do